woocommerce/woocommerce-ios
# Stats: add to the Storage layer for new stats

Fixes #1065 

Submitting as a draft PR, as it depends on PR #1082 to be reviewed and merged.

This PR adds stats V4 support to model v 17:

<img src="https://user-images.githubusercontent.com/2722505/61001130-025cc200-a32d-11e9-99c9-269e1a09d100.png" width="350"/>

## Changes
- New model version, with new entities.
- Generated subclasses
- Added a convenience method to the core data extensions

## Testing
- Checkout the branch, build and run the app, verify there are no coredata migration errors
- Run the test, verify they are ✅
